examples/README.rst

```diff
@@ -1,34 +0,0 @@
-========
```

Comment:
Why delete this?

Reply:
Ummm not sure, just removed this unused directory... Maybe this was still here by mistake?
@ItayGabbay ? Is this needed?

Reply:
This was here on purpose - so people looking for examples will know where to go. This readme points them to the docs.

Reply:
Re-added